src/darwin/Framework/CHIP/templates/CHIPClustersObjc-src.zapt

```diff
@@ -118,6 +133,21 @@ using namespace chip::app::Clusters;
 }
 - (void)write{{>attribute}}WithValue:({{asObjectiveCType type parent.name}})value params:(CHIPWriteParams * _Nullable)params completionHandler:(StatusCompletion)completionHandler
 {
+    ListFreer listFreer;
+    using TypeInfo = {{asUpperCamelCase parent.name}}::Attributes::{{asUpperCamelCase name}}::TypeInfo;
+    TypeInfo::Type cppValue;
+    {{>encode_value target="cppValue" source="value" cluster=parent.name errorCode="return;" depth=0}}
+
+    {{#unless mustUseTimedInvoke}}
```

Comment:
`mustUseTimedInvoke` will never be true for a _write_.  But otherwise, similar comments as above.

Reply:
I remove the `mustUseTimedInvoke` filter.